lib/decent_exposure/exposure.rb

```diff
@@ -3,8 +3,8 @@ class Exposure
     attr_reader :controller, :options
 
     # Public: Initializes an Exposure and makes it accessible to a controller.
-    # For each Exposure, a getter and setter is defined.
-    # Those getters and setters are made available to
+    # For each Exposure, a getter is defined.
+    # Those getters are made available to
     # the controller as helper methods.
```

Comment:
For each Exposure, a getter and setter is defined.
Getter is made available to the controller as helper method.